core/templates/dev/head/pages/about/about.html

```diff
@@ -26,94 +26,59 @@
         <div class="about-nav md-tab">
           <ul class="oppia-about-tabs">
             <li class="oppia-about-tabs-active">
-              <a class="oppia-about-tabs-text" href="about">About</a>
+              <a class="oppia-about-tabs-text" href="about" translate="I18N_ABOUT_PAGE_TABS_ABOUT"></a>
             </li>
             <li>
-              <a class="oppia-about-tabs-text" href="foundation" name="foundation">Foundation</a>
+              <a class="oppia-about-tabs-text" href="foundation" name="foundation" translate="I18N_ABOUT_PAGE_TABS_FOUNDATION"></a>
             </li>
             <li>
-              <a class="oppia-about-tabs-text" href="credits" name="credits">Credits</a>
+              <a class="oppia-about-tabs-text" href="credits" name="credits" translate="I18N_ABOUT_PAGE_TABS_CREDITS"></a>
             </li>
           </ul>
         </div>
 
         <div class="about-tab-container">
           <div class="about oppia-about-tab-content oppia-about-visible-content">
             <div class="oppia-static-card-content oppia-static-card-content-narrow">
-              <h2>About Oppia</h2>
+              <h2 translate="I18N_ABOUT_PAGE_ABOUT_TAB_HEADING"></h2>
               <div class="oppia-about-text-align">
                 <em>Oppia "O-pee-yah" (Finnish) - "to learn"</em>
               </div>
-              
-              <p>
-                Oppia's mission is to help anyone learn anything
-                they want in an effective and enjoyable way.
+
+              <p translate="I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_1">
```

Comment:
nit: I think this paragraph tag doesn't have a closing tag? It's not necessary, but aids consistency and maintainability (and other paragraph tags on the page do have closing tags).